system/indy-node-tests/test_state_proof.py

```diff
@@ -196,3 +207,28 @@
     res10 = json.loads(await ledger.submit_request(pool_handler, req10))
     print(res10)
     assert res10['result']['seqNo'] is not None
+
+    # no seqno returned for this txn type
+    req11 = await ledger.build_get_auth_rule_request(None, '101', 'ADD', '*', None, '*')
+    res11 = json.loads(await ledger.submit_request(pool_handler, req11))
+    print(res11)
```

Comment:
Do we still need those prints? I see that you're removing old ones, at the same time adding new prints with new functionality? I suspect there is some rationale behind this, is it?